docs/reference/authentication.md

```diff
@@ -14,6 +14,22 @@ We recommend using basic authentication and a strong password to protect the `/s
 
 The API Gateway as of version 0.8.2 provides built-in basic authentication. To use it set the environmental variable `basic_auth` to true. Then create two secrets named `basic-auth-user` and `basic-auth-password`.
 
+**To create the secrets there are two options:**
+
+Option 1 (simpler, but doesn't works under Windows and docker complains about security)
+```bash
+echo myusername | docker secret create basic-auth-user -
```

Comment:
Prepend lines that show commands with $
Do you find that echo appends a `\n`?  Is this desirable?